packages/form-js-editor/karma.conf.js

```diff
@@ -37,8 +37,10 @@ module.exports = function(karma) {
 
     coverageReporter: {
       reporters: [
-        { type: 'lcovonly', subdir: '.' },
-      ]
+        { type: 'lcovonly', subdir: '.' }
+      ].concat(process.env.HTML_COVERAGE ? [
+        { type: 'html', subdir: '.' }
+      ] : [])
```

Comment:
Maybe to clarify: why do we use `lcovonly` at all? In bpmn-js [we use `lcov`](https://github.com/bpmn-io/bpmn-js/blob/develop/test/config/karma.unit.js#L46) which offers HTML reporting as well. 

Is it to reduce the overhead when dealing with more packages?

Reply:
No, there is literally no reason, other than that is how we do it.

If lcov also generates HTML reports, let us simply use it :).

Reply:
Will double check and update PR accordingly.

Reply:
Updated via https://github.com/bpmn-io/form-js/pull/250/commits/c25d14d0231dbea4b4cc08641669867d49b5e837.